dev_guide/secrets.adoc

```diff
@@ -116,6 +116,20 @@ Pull Secrets] for more information.
 See link:builds.html#using-private-repositories-for-builds[Using Private
 Repositories for Builds] for more information.
 
+[[service-serving-certificate-secrets]]
+=== Service Serving Certificate Secrets
+
+To secure communication to your service, you can have the cluster generate a signed
+serving certificate/key pair into a secret in your namespace.  To do this, set the
+"service.alpha.openshift.io/serving-cert-secret-name" to the name you want to use
+for your secret.  Your PodSpec can then mount that secret and when it is available
+your pod will run.  The certificate will be good for the internal service DNS name:
```

Comment:
> How long is the cert valid for, and how should renewal be handled?

One year.  The controller could be tweaked to find certs that are close to expiry and regenerate them.  It hasn't been done yet, but the clock is ticking.